css/portal-dashboard/progress-view-legend.less

```diff
@@ -79,3 +78,24 @@
     }
   }
 }
+@media screen and (max-width: 1000px) {
+  .legendContainer {
+    line-height: 25px;
+    bottom: -30px;
+    justify-content: center;
+    flex-wrap: wrap;
+    width: 60%;
+    min-width: 450px;
+    left: calc(((100vw - @left-panel-width) / 2) - @left-panel-width);
```

Comment:
I would think that you don't need 100vw to get a left position. Isn't it a constant?

Reply:
Or is this to center it?

Reply:
In which case you'll get a better result with:
`left: calc(((100vw - 240px) / 2) - 225px);`

Reply:
since it's basically, ((screenWidth - leftPanelWidth) / 2) - (legendWidth / 2)
legendWidth / 2 is not the same as leftPanelWidth.

Reply:
It's to center it. Changing it so to legendWidth/3 seems to work better than legendWidth/2

Reply:
That shouldn't be the case.  Why divide it by 3?
Did you try `left: calc(((100vw - 240px) / 2) - 225px);`?

Reply:
Here is using my calculation:
![image](https://user-images.githubusercontent.com/5126913/105905393-37943800-5fd7-11eb-8d00-09f4bdeb6002.png)

Reply:
Dividing by 3 will move it too far to the right.  legendWidth is 450px so 225px is what should be used.

Reply:
Again, the standard mechanism for this type of centering when an object has left edge positioning is to take availableWidth / 2 - objectWidth / 2.

Reply:
Using that we get perfect centering:
![center](https://user-images.githubusercontent.com/5126913/105905858-d1f47b80-5fd7-11eb-89e9-b3a2a24bf4d9.gif)